tests/utils/test_extension_process_util.py

```diff
@@ -164,29 +164,26 @@ def test_handle_process_completion_should_raise_on_nonzero_exit_code(self):
                 self.assertEquals(context_manager.exception.code, error_code)
                 self.assertIn("Non-zero exit code:", ustr(context_manager.exception))
 
-    def test_read_output_it_should_return_no_content(self):
+    def test_read_output_should_return_no_content(self):
```

Comment:
Once I fixed the patching for `TELEMETRY_MESSAGE_MAX_LEN`, these tests produced different (correct) results :)